Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java

```diff
@@ -62,7 +62,7 @@
     @Autowired
     private NetworkDTOMapper networkMapper;
 
-    @Resource(name = "database_error_msgs")
+    @Resource(name = "projectconstants")
     private Properties messages;

```

Comment:
ah that makes sense. Would you please just delete this? It's in the parent class.